com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/Constants.java

```diff
@@ -0,0 +1,25 @@
+package com.sap.cloud.lm.sl.cf.persistence;
+
+public class Constants {
+    public static final String VARIABLE_NAME_SPACE_ID = "__SPACE_ID";
+    public static final String CORRELATION_ID = "correlationId";
+    public static final String TASK_ID = "__TASK_ID";
+    public static final String PROCESS_ABORTED = "__PROCESS_ABORTED";
+    public static final String INDEXED_STEP_NAME = "__INDEXED_STEP_NAME";
+    public static final String VARIABLE_NAME_SERVICE_ID = "__SERVICE_ID";
+
+    public static final int DEFAULT_BUFFER_SIZE = 4 * 1024;
+
+    public static final String CONTENT = "CONTENT";
+    public static final String MODIFIED = "MODIFIED";
+    public static final String DIGEST_ALGORITHM = "DIGEST_ALGORITHM";
+    public static final String FILE_SIZE = "FILE_SIZE";
+    public static final String NAMESPACE = "NAMESPACE";
+    public static final String SPACE = "SPACE";
+    public static final String FILE_NAME = "FILE_NAME";
+    public static final String DIGEST = "DIGEST";
+    public static final String FILE_ID = "FILE_ID";
+
+    protected Constants() {
```

Comment:
private constructor?

Reply:
The constructor is protected because this class is overrided